packages/contracts-bedrock/src/L2/SoulGasToken.sol

```diff
@@ -80,6 +80,16 @@ contract SoulGasToken is ERC20Upgradeable, OwnableUpgradeable {
         require(msg.value == totalValue, "unexpected msg.value");
     }
 
+    /// @notice batchDepositForAll is similar to batchDepositFor, but the value is the same for all accounts.
+    function batchDepositForAll(address[] calldata accounts, uint256 value) external payable {
```

Comment:
Do we want to follow the OP's coding style here?

Reply:
Good catch, fixed!